common/lib/api-client/middleware/get-cache.js

```diff
@@ -0,0 +1,30 @@
+const debug = require('debug')('app:api-client:get-cache')
+
+const cache = require('../cache')
+
+function getCacheMiddleware({ useRedisCache = false } = {}) {
```

Comment:
Same with this default. I think if we want a _default_ that value should live in the configuration and be passed into this function.

Reply:
As per https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/801#discussion_r492632004

Should it throw if not passed a defined value?

Reply:
Same response here I'd say. I don't think this function needs to be that aggressive. If we want a default it should be set in the config and then there is no case of it not having a value.

The reason for passing that into this function here is to make it a little more pure so that it doesn't need to reach into the config, so I think it's okay to not throw in this instance.

Reply:
in which case there's little difference between `{ useRedisCache = false }` and `{ useRedisCache }` other than necessitating an extra test to check that undefined behaves the same as false